src/ILLink.RoslynAnalyzer/RequiresAssemblyFilesAnalyzer.cs

```diff
@@ -95,6 +152,39 @@ public override void Initialize (AnalysisContext context)
 							url));
 					}
 				}
+
+				static bool Contains<T, TComp> (ImmutableArray<T> list, T elem, TComp comparer)
```

Comment:
I think we should move this and bellow methods to another file containing helper methods.